### ISSUE TITLE
option for basic GUI to select run mode

### ISSUE THREAD

**Issue by privatezero** [amiaopensource/vrecord#85]
In discussions with people who use vrecord in a community archiving environment, it was mentioned that a centralized gui for selecting the run mode would help with learning curve for people less familiar with CLI.  Gonna see if something like zenity could help with this!

**Comment by privatezero** [comment]
hey @libbyhopfauf!  Trying out a secondary GUI that can be accessed with the command vrecord -g (so you could just create one desktop script instead of one for each option) Looks like this so far. Thoughts? 
<img width="333" alt="vrecord_central_gui" src="https://cloud.githubusercontent.com/assets/12941699/20322966/f34b3672-ab48-11e6-8cbb-2208bb3b015c.png">

**Comment by dericed** [comment]
Maybe `edit` should be renamed to `edit preferences` to give more context.

**Comment by privatezero** [comment]
As the envisioned use case for this is to help set up stations for people less familiar with CLIs and A/V issues do you think any others should be renamed to more explicit natural language (i.e. passthrough becoming something like viewing mode?).  

Also- Thanks for the logo @libbyhopfauf!!

**Comment by dericed** [comment]
fixed in https://github.com/amiaopensource/vrecord/pull/107

**Comment by libbyhopfauf** [comment]
Thanks for working on this, Andrew! I wanted to have several test subjects (who are the intended audience i.e. less familiar with CLIs and A/V issues) try out the GUI mode and it took long than I thought for everyone to have time. They all commented that they would prefer the interface to be even more simplified to make it the most efficient. From their feedback, I was thinking of something more like this:

![screen shot 2016-12-07 at 6 59 06 pm](https://cloud.githubusercontent.com/assets/13877631/20996089/e80b1386-bcaf-11e6-81ba-dce3992fc29b.png)

However, since I took so long to respond and since I am not sure how many people would actually use this/have need of it, I created a folder with the commands/icons in it that basically runs the same way :)
